src/SharedModule/Components/HeaderComponents/HeaderComponents.tsx

```diff
@@ -5,6 +5,8 @@ type HeaderProps = {
   buttonName: string;
   addOn: 'yes' | 'no'; 
   anyFunction: () => void; 
+  OpenFacilitiesAddModal:Function;
+  listName:string;
 }
 
 const theme = createTheme();
```

Comment:
The theme should be in a top level component like the App , and provide it through the ThemeProvider. This will enable us to access it from any descendant components without redeclaring it again.